legacy/firmware/usb.h

```diff
@@ -25,8 +25,32 @@
 void usbInit(void);
 void usbPoll(void);
 void usbReconnect(void);
+
+/*
+ * Setting this value to 0 will limit the protobuf messages `usbPoll` and
+ * `usbSleep` can handle to a few defined in `msg_read_tiny`.
+ *
+ * Setting to 0 is meant to prevent infinite recursion when you need to read a
+ * message while being called from FSM.
+ *
+ * Setting to 1 allows processing all messages.
+ */
 char usbTiny(char set);
-void usbSleep(uint32_t millis);
+
+/*
+ * This will wait given number of milliseconds for arrival of protobuf message.
+ * If it arrives, it will service it before returning.
+ *
+ * If you call this function from any function that is called from FSM,
+ * you must use `usbTiny(0)` before and `usbTiny(oldTinyValue)` after, otherwise
```

Comment:
I think this is not correct.

The pattern referenced here is as follows:
```C
oldTinyValue = usbTiny(1);
// do an operation that requires reading USB messages of the "tiny" type
usbTiny(oldTinyValue)
```

Technically, 
```C
usbTiny(1);
// do an operation that requires reading USB messages of the "tiny" type
usbTiny(0)
```
also works, but has different side effects on the global tiny state after processing.

In either case, a `usbTiny(1)` is required to go into the special mode, not `usbTiny(0)`.